website/src/css/custom.css

```
@@ -0,0 +1,225 @@
+  background: var(--ifm-color-primary-dark);
+  color: var(--yv-on-primary);
+  text-decoration: none;
+}
+
+/*
+ * The glyph is drawn with a mask so it takes `currentColor` from the button
+ * instead of shipping one image per theme.
+ */
+.navbar-github {
+  --yv-btn-fg: var(--ifm-color-emphasis-800);
+  margin-left: 0.5rem;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)